core/templates/pages/topic-viewer-page/practice-tab/practice-tab.component.html

```diff
@@ -100,9 +102,17 @@ <h2 class="tab-title" *ngIf="displayArea === 'topicViewer'">
     border-bottom: 1px solid;
   }
 
+  .practice-buttton-background-status-complete {
+    background-color: #00645C;
+  }
+
+  .practice-buttton-background-status-pending {
+    background-color: rgba(0, 0, 0, .26);
```

Comment:
please try using hex color code.

Reply:
We used RGBA just because of it's opacity parameter, however if we use hex value then we have to define background and opacity as a separate property like :
{
    background: # 000000;
    opacity: 0.2;
}
Apart from this we can also use 8-character hex value, but this is very uncommon.

Reply:
Ah i just saw the documentation guidelines, both rgba and hex both can work. Sorry about this one.